build/nuget-release.yml

```diff
@@ -0,0 +1,172 @@
+name: $(Package.Version)
+
+trigger: none
+pr: none
+
+resources:
+  repositories:
+    - repository: templates
+      type: github
+      name: arcus-azure/azure-devops-templates
+      endpoint: arcus-azure
+
+variables:
+  - group: 'Build Configuration'
+  # Always use fixed version for .NET Core SDK
+  - name: 'DotNet.Sdk.Version'
+    value: '2.2.402'
+  - name: 'Project'
+    value: 'Arcus.Template'
+  - name: 'Http.Port'
+    value: '5000'
+  - name: 'Arcus_Api_BaseUrl'
+    value: 'http://localhost:$(Http.Port)/api/v1'
+  # 'Package.Version.ManualTrigger' is added as queue-time variable on build in Azure DevOps
+
+stages:
+  - stage: Build
+    jobs:
+      - job: Compile
+        pool:
+          vmImage: 'ubuntu-16.04'
+        steps:
+          - task: DotNetCoreInstaller@0
+            displayName: 'Import .NET Core SDK ($(DotNet.Sdk.Version))'
+            inputs:
+              version: '$(DotNet.Sdk.Version)'
+          - task: DotNetCoreCLI@2
+            displayName: 'Compile'
+            inputs:
+              projects: 'src/*.sln'
+              arguments: '--configuration release /property:Version=$(Build.BuildNumber)'
+          - task: CopyFiles@2
+            displayName: 'Copy build artifacts'
+            inputs:
+              contents: '**/bin/**/*.nupkg'
+              targetFolder: '$(Pipeline.Workspace)/templates'
+              flattenFolders: true
+          - task: PublishPipelineArtifact@0
+            displayName: 'Publish build artifacts'
+            inputs:
+              targetPath: '$(Pipeline.Workspace)/templates'
+              artifactName: Templates
+
+  - stage: IntegrationTests
+    displayName: Integration Tests
+    dependsOn: Build
+    condition: succeeded()
+    jobs:
+      - job: RunIntegrationTests
```

Comment:
Thinking out loud and beyond scope of this PR, but would job templates make sense?
https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema#job-templates

However, I'm wondering if we should run all tests here given we've already tested things, but can't hurt - Right?!

Reply:
If so, let's create an issue for it

Reply:
Yeah, job templates could maybe replace some 'basic' steps which are duplicated.